Neos.Fusion/Classes/Core/Runtime.php

```diff
@@ -148,15 +154,7 @@ public function __construct(
         );
         $this->runtimeContentCache = new RuntimeContentCache($this);
         $this->fusionGlobals = $fusionGlobals;
-    }
-
-    /**
-     * @deprecated {@see self::getControllerContext()}
-     * @internal
-     */
-    public function setControllerContext(ControllerContext $controllerContext): void
```

Comment:
this method was only introduced in 9.0 dev thats why we can so easily remove it: https://github.com/neos/neos-development-collection/pull/4425